docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -577,7 +577,92 @@ You should be redirected again to the login page at Keycloak. However, now you a
 In both cases, if the user is successfully authenticated, the landing page will show the user's name and e-mail. Even though
 user `alice` exists in both tenants, for the application they are distinct users belonging to different realms/tenants.
 
-== Resolving Tenant Identifiers with Annotations
+[[static-tenant-resolution]]
+== Static Tenant configuration resolution
+
+If you have prepared multiple tenant configurations in `application.properties` then it is only a tenant identifier
+that has to be resolved and you can use one of following 3 options to resolve tenant identifiers:
+
+[[default-tenant-resolver]]
+=== Default Resolution
+
+Default tenant identifier resolution is convention based, it expects requests requiring authentication to have a tenant identifier set as the last request path segment value.
+
+Lets assume tenant `google` and `github` configurations are set in `application.properties`:
+
+[source,properties]
+----
+# Tenant 'google' configuration
+quarkus.oidc.google.provider=google
+quarkus.oidc.google.client-id=${google-client-id}
+quarkus.oidc.google.credentials.secret=${google-client-secret}
+quarkus.oidc.google.authentication.redirect-path=/signed-in
+
+# Tenant 'github' configuration
+quarkus.oidc.github.provider=google
+quarkus.oidc.github.client-id=${github-client-id}
+quarkus.oidc.github.credentials.secret=${github-client-secret}
+quarkus.oidc.github.authentication.redirect-path=/signed-in
+----
+
+These 2 tenants configure OIDC `web-app` applications which will use an authorization code flow to authenticate users and will depend on session cookies after the authentication has taken place. In both cases, after either Google or GitHub have authenticated the current user, the user is returned to the `/signed-in` area for the already authenticated users - for example, it can be a secured resource path on the JAX-RS endpoint.
+
+Now, the only thing that you will need to make the resolution working by default is to add the following:
+
+[source,properties]
+----
+quarkus.http.auth.permission.login.paths=/google,/github
+quarkus.http.auth.permission.login.policy=authenticated
+----
+
+and, assuming the endpoint is running on `http://localhost:8080`, provide UI options for users to login to either `http://localhost:8080/google` or `http://localhost:8080/github`, without having to add JAX-RS `/google` or `/github` JAX-RS resource paths. Tenant identifiers will also be recorded in the session cookie names after the authentication has been completed and therefore authenticated users will be able to access the secured application area without secured URLs having to contain either `google` or `github` path values.
+
+Default resolution can also work for Bearer token authentication but it may be less practical in this case since a tenant identifier will always have to be set as the last path segment value.
```

Comment:
```suggestion
Default resolution can also work for Bearer token authentication but it might be less practical in this case because a tenant identifier will always need to be set as the last path segment value.
```